benchmarks/dynamo/operatorbench/operators/FusedLinearCrossEntropy/inductor.py

```diff
@@ -0,0 +1,26 @@
+from utils.common import BenchmarkConfig
+
+import torch
+
+from . import FusedLinearCrossEntropyOperator, H, V
+from .baseline import TorchLMHeadCE
+
+
+class TorchLMHeadCECompiled(TorchLMHeadCE):
+    def __init__(self, H: int, V: int, dtype: torch.dtype, ignore_index: int = -100):
+        super().__init__(H, V, dtype, ignore_index)
+        self.lin = torch.compile(self.lin)
+        self.ce_loss = torch.compile(self.ce_loss)
```

Comment:
You should call `torch.compile` on the entire model. Also can the torch.compile variant being generated automatically from the base version rather than requiring adding a file manually?

Reply:
fixed in https://github.com/pytorch/pytorch/pull/136169/commits/7c2bc74a72652a45e0f4a872958d611c43ba23bd. I'll do the automatically torch.compile for native pytorch implementation. For custom kernels, it can be hard to write a general way because they may have quite different implementations. maybe we can provide an API or decorator for users to add the inductor variant.